### PR TITLE
feat: updated jq with jmespath

### PR DESCRIPTION
**Pull request by Blokh**
### Description
updated jq with jmespath in order to be available in web sdk

### CODE REVIEW

File: packages/workflow-core/package.json [hunk 1]
```diff
@@ -30,9 +30,10 @@
   },
   "dependencies": {
     "@ballerine/common": "0.5.3",
+    "@types/jmespath": "^0.15.0",
```
Comment (Omri-Levy) [inline packages/workflow-core/package.json:33]: Make sure `@types/...`s are in `devDependencies`.

File: packages/workflow-core/src/lib/utils/context-transformers/qj-transformer.ts [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-    this.mapping = mapping;
-  }
-
-  async transform(context: TContext, options: {}) {
-    const response = await run(this.mapping, context, options);
-
-    return response;
-  }
-}
+// import { run } from 'node-jq';
```
Comment (Omri-Levy) [inline packages/workflow-core/src/lib/utils/context-transformers/qj-transformer.ts:1]: Do we still need this file?

File: services/workflows-service/package.json [hunk 3]
```diff
@@ -28,10 +28,10 @@
     "db:init": "npm run db:migrate-save -- --name 'initial version' && npm run db:migrate-up seed",
     "prisma:generate": "prisma generate",
     "json-schema:generate": "json2ts src/workflow/schemas/default-context-schema.json src/workflow/schemas/context.d.ts",
-    "docker:db": "docker compose -f docker-compose.db.yml up -d --wait",
+    "docker:db": "docker-compose -f docker-compose.db.yml up -d --wait",
```
Comment (Omri-Levy) [inline services/workflows-service/package.json:31]: Revert `docker-compose` to `docker compose` please. 🙏

File: packages/workflow-core/src/lib/utils/context-transformers/qj-transformer.ts (hunk 2 above)
Comment (Blokh) [reply packages/workflow-core/src/lib/utils/context-transformers/qj-transformer.ts:1]: not sure about it. might be injected later on. 
for now i see no harm of keeping it as a snippet

**Review by alonp99 — APPROVED** [review]

**Review by alonp99 — APPROVED** [review]

**Pull request merged**